### ISSUE TITLE
Feature:  1.0.14 Reference Section Preferences

### ISSUE THREAD

**Issue by pkbott** [mpogue2/SquareDesk#1243]
Is there a way to make the zoom preferences and other preferences stick in the reference section?  I like the fit to width zoom option and it always resets to automatic zoom.  Just wondering if this is possible in the program or not?

**Comment by mpogue2** [comment]
Good question -- I don't know.  This one would take some digging...

**Comment by mpogue2** [comment]
OK I did the digging, and there's good news and bad news:

**Good:** The latest version (v4) of the pdf.js library (which SquareDesk uses to display PDF's) does seem to support saving the preferences, which is exactly what you are asking for.
**Bad:** I can't for the life of me figure out how to get pdf.js v4 to work in SquareDesk.  They seem to have changed a bunch of things that cause it to not work properly when I try it.  I tried for a couple of hours and I have given up for now.  I am not sure what I'm doing wrong. 

Bottom line: It seems there should be a way to make this work, but it may take a bunch more hours to figure out.
It's a good feature request, so it should be done, I just don't if I am smart enough to do it! :-)

**Comment by mpogue2** [comment]
The method I used before was this:
https://icrunchdata.com/blog/584/power-of-qt-making-a-pdf-viewer-desktop-application-in-a-few-hours/
but, the work was done for me then.  I can't get the v4 pdf.js to build, and I can't seem to get gulp installed either.
But, this is a clue as to how it was done earlier.

**Comment by mpogue2** [comment]
I think I might at this point go with the approach described in #648, where we swap out the pdf.js engine, and swap in the relatively new (but fast) QPdfView engine.  Qt Creator has both pdfViewer and a documentViewer that show how to use the new engine.  Then we could make the selection persistent, fairly easily.
